numpy/_core/src/multiarray/methods.c

```diff
@@ -2756,6 +2756,33 @@ array_class_getitem(PyObject *cls, PyObject *args)
     return Py_GenericAlias(cls, args);
 }
 
+static PyObject *
+array_array_namespace(PyArrayObject *self, PyObject *args, PyObject *kwds)
+{
+    static char *kwlist[] = {"api_version", NULL};
+    char *array_api_version = "2022.12";
+
+    if (!PyArg_ParseTupleAndKeywords(args, kwds, "|$s:__array_namespace__", kwlist,
+                                     &array_api_version)) {
+        return NULL;
+    }
+
+    if (strcmp(array_api_version, "2021.12") != 0 &&
```

Comment:
Is the point of array_api_version to be forwards compatible, or should this error too?

Reply:
2022.12 is a superset of 2021.12, so if a user asks for the latter, it's fine to return the former.